CONTRIBUTING.md

```diff
@@ -139,8 +137,19 @@ Typedoc generates this documentation from the main [firebase index.d.ts type def
 
 If any pages are added or removed by your change (by adding or removing a class or interface), the [js/toc.yaml](scripts/docgen/content-sources/js/toc.yaml) and/or [node/toc.yaml](scripts/docgen/content-sources/node/toc.yaml) need to be modified to reflect this.
 
+# Formatting Code
+A Formatting Check CI failure in your PR indicates that the code does not follow the repo's
+formatting guidelines. In your local build environment, please run the code formatting tool locally
+by executing the command $ yarn format. Once the code is formatted, commit the changes and push your
```

Comment:
backticks are more standard for something to run in the command line instead of $?